axelor-account/src/main/resources/i18n/messages_ru.csv

```diff
@@ -1377,7 +1377,6 @@
 "Fixed asset type","Тип основных средств",,
 "Fixed asset types","Виды основных средств",,
 "Fixed asset: could not generate depreciation move because fixed category is one of missing theses account %s",,,
-"Fixed asset: disposal move could not be generated because fixed  category is missing one of theses account %s",,,

```

Comment:
Please don't update these files, just update messages.csv, messages_en.csv and messages_fr.csv